src/networkmanager.cpp

```diff
@@ -57,8 +58,8 @@ void NetworkManager::connect_to_server(server_type p_server)
 {
   server_socket->close();
   server_socket->abort();
-
-  server_socket->connectToHost(p_server.ip, p_server.port);
+  QHostAddress addr(p_server.ip);
+  server_socket->connectToHost(addr, p_server.port);
```

Comment:
This should not make any difference. See the source for [`void QAbstractSocket::connectToHost(const QHostAddress &address, quint16 port, OpenMode openMode)`](https://code.woboq.org/qt5/qtbase/src/network/socket/qabstractsocket.cpp.html#_ZN15QAbstractSocket13connectToHostERK12QHostAddresst6QFlagsIN9QIODevice12OpenModeFlagEE).

Reply:
Correct, tested without this and it works either way. Removed as of `8932a30`